chart/compass/charts/external-services-mock/templates/auditlog-configmap.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.global.auditlog.configMapName }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ .Chart.Name }}
+    release: {{ .Release.Name }}
+data:
+  auditlog-oauth-user-var: "mock-user"
+  auditlog-oauth-tenant-var: "mock-tenant"
+  auditlog-url: "http://compass-external-services-mock.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.service.port }}"
```

Comment:
`auditlog-oauth-tenant-var` and `auditlog-oauth-user-var` is not needed.
We have default values, which are checked in `external-services-mock`.